Common/src/grid_movement_structure.cpp

```diff
@@ -127,16 +125,13 @@ void CVolumetricMovement::UpdateMultiGrid(CGeometry **geometry, CConfig *config)
 
 void CVolumetricMovement::SetVolume_Deformation(CGeometry *geometry, CConfig *config, bool UpdateGeo, bool Derivative) {
 
-  unsigned long IterLinSol = 0, Smoothing_Iter, iNonlinear_Iter, MaxIter = 0, RestartIter = 50, Tot_Iter = 0, Nonlinear_Iter = 0;
-  su2double MinVolume, MaxVolume, NumError, Residual = 0.0, Residual_Init = 0.0;
-  bool Screen_Output;
+  unsigned long Tot_Iter = 0;
+  su2double MinVolume, MaxVolume;
 
   /*--- Retrieve number or iterations, tol, output, etc. from config ---*/
 
-  Smoothing_Iter = config->GetDeform_Linear_Solver_Iter();
-  Screen_Output  = config->GetDeform_Output();
-  NumError       = config->GetDeform_Linear_Solver_Error();
-  Nonlinear_Iter = config->GetGridDef_Nonlinear_Iter();
+  auto Screen_Output  = config->GetDeform_Output();
```

Comment:
This is a matter of taste, but in situations like this where it isn't clear what the type being returned is from the method name alone (or if it doesn't save you a ton of keystrokes), I would avoid auto and stick with bool to be more clear.